ui/admin/app/routes/scopes/scope/targets.js

```diff
@@ -51,16 +51,21 @@ export default class ScopesScopeTargetsRoute extends Route {
   /**
    * Handle save.
    * @param {TargetModel} target
-   * @param {Event} e
+   * @param {object} options
    */
   @action
   @loading
-  @notifyError(({ message }) => message)
+  @notifyError(({ message }) => message, { catch: true })
```

Comment:
When should we be squelching errors? After I curry this save function in the form, I started to get errors in tests from the adapter which I couldn't catch, is there a better way of solving this?